### PR TITLE
Add support for Salus FC600

### PR DESCRIPTION
**Pull request by mkrle**
Feedback welcome!

TODO:

- `running_state` will report values not in `thermostatRunningStates` constants if the fan is running on med or high speed. Should I add those states to `thermostatRunningStates` in `src/lib/constants.ts` or is there a better way?
- OTA fails with `device.definition.ota.isUpdateAvailable is not a function`. I haven't been able to figure this out so far. If I can't make it work I'll remove it from the definition.
- I don't know how the "presets" like eco/off etc are configured. I think it's nice to have but I don't see this as a blocker

### CODE REVIEW

File: src/devices/salus_controls.ts [hunk 1]
```diff
@@ -95,6 +96,55 @@ const definitions: DefinitionWithExtend[] = [
...
+            const binds = ['genBasic', 'genIdentify', 'genTime', 'hvacThermostat', 'hvacFanCtrl', 'hvacUserInterfaceCfg'];
+            await reporting.bind(endpoint, coordinatorEndpoint, binds);
+            await reporting.thermostatTemperature(endpoint);
+            await reporting.thermostatOccupiedHeatingSetpoint(endpoint);
+            await reporting.thermostatSystemMode(endpoint);
+            await reporting.fanMode(endpoint);
+            await reporting.thermostatRunningMode(endpoint);
+            await reporting.thermostatRunningState(endpoint);
+        },
+        ota: {manufacturerName: 'SalusControls'},
```
Comment (Koenkk) [inline src/devices/salus_controls.ts:146]:
```suggestion
        ota: true,
```

**Comment by Koenkk** [comment]
> running_state will report values not in thermostatRunningStates constants if the fan is running on med or high speed. Should I add those states to thermostatRunningStates in src/lib/constants.ts or is there a better way?

Please add them there

**Comment by mkrle** [comment]
> Please add them there

Thanks, done.

**Comment by Koenkk** [comment]
I will squash them, thanks!

**Pull request merged**